tests/labeler/chexpert/config_test_label.py

```diff
@@ -0,0 +1,58 @@
+import csv
```

Comment:
test fies always have to start with "test_....py"

Reply:
sorry, didn't think that's also applicable for the config, will change it asap :)